.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+---
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12, 14]
+        package: ["react", "widgets"]
```

Comment:
The `core` & `webcomponents` packages are critical IMO: the first is the common dependency across all SDK and the `webcomponents`'s latest stable version is Builder's main entrypoint for all Shopify content.

We could start with a try-to-build CI step?

Reply:
Done!